index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8">
+    <title> How to view the HTML source of a Website on Chrome </title>
+    <link rel="stylesheet" href="./styles.css">
+  </head>
+
+  <body>
+<h1> How to view the HTML source of a Website on Chrome </h1>
+<img src='./img.png' alt='View Source Code picture' width="800" height="500">
+
+<h2> There are 4 steps you need to take in order to "reveal the magician's secrets"</h2>
+
+<ol>
+  <li>Open the browser (Google Chrome).</li>
+  <li>Go to the page you wish to see the HTML code for.</li>
+  <li>Right-click on the page.</li>
+  <li>Click 'View page source.'</li>
+</ol>
+
+<table style="width:100%">
+  <tr class='blue'>
+    <th>Monday</th>
+    <th>Tuesday</th>
+    <th>Wednesday</th>
+    <th>Thursday</th>
+    <th>Friday</th>
+    <th>Saturday</th>
+    <th>Sunday</th>
+  </tr>
+  <tr class='red'>
+    <td>10:00 AM: Wake Up</td>
```

Comment:
Design-wise, you probably want it to look like:

|      | Monday | Tuesday | Wednesday | ... |
| ---- | ------ | ------- | --------- | --- |
| 10AM | ...    | ...     | ...       | ... |
| 12PM | ...    | ...     | ...       | ... |
|  2PM | ...    | ...     | ...       | ... |
|  ... | ...    | ...     | ...       | ... |